claim-check/claim-check-consumer.bal

```diff
@@ -0,0 +1,30 @@
+import ballerinax/kafka;
+import ballerina/http;
+import ballerina/io;
+
+const MRI_SCANS = "mri-scans";
+
+final http:Client awsS3Client = check new ("http://bucket.s3.amazonaws.com.balmock,io");
+final http:Client firebaseClient = check new ("http://api.mriresults.firebase.com.balmock.io");
+final kafka:Consumer kafkaConsumer = check new (kafka:DEFAULT_URL, {
+    groupId: "mri-scan-group",
+    topics: [MRI_SCANS]
+});
+
+public function main() returns error? {
+    while true {
+        [string, string][] mriScanResults = check kafkaConsumer->pollPayload(1);
+        from [string, string] [mriScanFileName, patientId] in mriScanResults
+        do {
```

Comment:
lets use foreach and use `var` instead `[string, string]`

Reply:
We cannot use var here because we cannot infer the type here.